### PR TITLE
x11-apps/xrectsel: bump to 0.3.2

### PR DESCRIPTION
**Pull request by lolilolicon**

### CODE REVIEW

File: x11-apps/xrectsel/xrectsel-0.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (doughdemon) [inline x11-apps/xrectsel/xrectsel-0.3.2.ebuild]: Please run repoman and fix its warnings.

File: x11-apps/xrectsel/xrectsel-0.3.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (doughdemon) [inline x11-apps/xrectsel/xrectsel-0.3.2.ebuild]: Please update to EAPI 6

File: x11-apps/xrectsel/xrectsel-0.3.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+inherit autotools
+
+DESCRIPTION="Print the geometry of a rectangular screen region"
+HOMEPAGE="https://github.com/lolilolicon/xrectsel"
+SRC_URI="https://github.com/lolilolicon/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (doughdemon) [inline x11-apps/xrectsel/xrectsel-0.3.2.ebuild]: New versions should not be committed straight to stable.

File: x11-apps/xrectsel/xrectsel-0.3.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+IUSE=""
+
+DEPEND="x11-libs/libX11"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	eautoreconf
```
Comment (doughdemon) [inline x11-apps/xrectsel/xrectsel-0.3.2.ebuild]: Please produce proper releases in future. (Upload the result of `make dist`.)

**Comment by lolilolicon** [comment]
@zx2c4 as the maintainer, please help me out here.
I edited this manually as I'm not running Gentoo. My apologies for the trouble.

**Pull request closed without merging**